# **Reproducibility of AIRR_sequencing tools**

## **About the project**
The advent of Adaptive Immune Receptor Repertoire (AIRR) sequencing has revolutionized our understanding of the immune system’s diversity and composition. This powerful technique has enabled the exploration of the relatedness between immune receptor sequences, revealing the adaptive immune system’s capacity to generate a vast array of receptors that can bind to a wide variety of antigens. However, the reproducibility of AIRR-seq data analysis remains a critical challenge, influenced by pre-sequencing factors such as library preparation, sequencing protocol the platform used and bioinformatics tools used for analysis. These tools must correctly identify reads aligned to VDJ genes, clonotypes, and the repertoire’s response to specific antigens. Moreover, they must generate consistent and reliable results. 

This study evaluates the reproducibility of AIRR-seq bioinformatics tools by generating synthetic replicates through in-silico shuffling of sequencing reads. We focus on two tools, MiXCR and TRUST4, assessing their performance in clonotype identification and consistency of results across computational replicates. Synthetic replicates, created by shuffling the reads of fastq files, provide a controlled environment to test the reproducibility of bioinformatics tools, reducing the experimental variations that typically affect sequencing outputs. Using a dataset comprising publicly available TCR repertoires and in-house samples, we generated 10 synthetic replicates for each sample and analyzed them using MiXCR and TRUST4.


## **Pre-requisities**

## **Contents of the repository**

## **Introduction**

## **Analysis**

## **Results**

## **Conclusion**
